WebDriverAgentLib/Commands/FBElementCommands.m

```diff
@@ -181,6 +181,11 @@ + (NSArray *)routes
   NSInteger elementID = [request.parameters[@"id"] integerValue];
   XCUIElement *element = [elementCache elementForIndex:elementID];
   NSError *error = nil;
+  if (!element.fb_isVisible) {
+    if (![element fb_scrollToVisibleWithError:&error]) {
```

Comment:
Ugh yea sorry this shouldnt be part of this pull request but now that its come up, do you think it makes sense to scroll to element before doing the "click" command? In the deprecated UIA Framework this was the behavior, also according to the webdriver [spec](https://www.w3.org/TR/webdriver/#dfn-element-click), element click is to scroll to the element into view before clicking.